go/vt/sqlparser/ast_test.go

```diff
@@ -768,77 +771,6 @@ func TestSplitStatements(t *testing.T) {
 	}
 }
 
-func TestSplitStatementToPieces(t *testing.T) {
-	testcases := []struct {
-		input  string
-		output string
-	}{{
-		input:  "select * from table1; \t; \n; \n\t\t ;select * from table1;",
-		output: "select * from table1;select * from table1",
-	}, {
```

Comment:
is this covered with other test or this is moved to different file?

Reply:
https://github.com/vitessio/vitess/pull/17946#discussion_r2018008188